distributed/comm/utils.py

```diff
@@ -19,6 +21,63 @@
     OFFLOAD_THRESHOLD = parse_bytes(OFFLOAD_THRESHOLD)
 
 
+def is_msgpack_serializable(msg) -> bool:
+    """Check if a message is msgpack serializable
+
+    Parameters
+    ----------
+    msg: Any
+        Message to check
+    """
+    typ = type(msg)
+
+    # Only lists and dicts can contain non-msgpack-serializable objects
+    if typ is not list and typ is not dict:
+        return True
+
+    if len(msg) == 0:
+        return True
+
+    if typ is not list:
+        msg_list = (msg,)
+    else:
+        msg_list = msg
+
+    for msg in msg_list:
+        if type(msg) is not dict:
+            return False
+
+        if set(msg.keys()) == {"compression", "python", "pickle-protocol"}:
+            continue
+
+        op = msg.get("op", None)
+        if not op:
+            return False
+
+        if "task" in msg:
+            return False
+        if "data" in msg:
+            return False
+        if msg.get("status", "OK") != "OK":
+            return False
+        if len(msg.get("worker-plugins", ())) > 0:
+            return False
+        if msg.get("actors", ()):
+            return False
+
+        # Operations explicit disallowed
+        if op in (
+            "actor_execute",
+            "actor_attribute",
+            "plugin-add",
+            "proxy",
+            "task-erred",
+        ):
```

Comment:
Thoughts on putting this list in a global variable above. That may make easy to add to as well